### PR TITLE
[cypher/*] Update cypher for camelCase vs PascalCase usage

### PR DESCRIPTION
**Pull request by encX**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: cypher.html.markdown [hunk 1]
```diff
@@ -50,7 +50,7 @@ Relationships (or Edges)
 **Connects two nodes**
 
 `[:KNOWS]`
-It's a *relationship* with the *label* **KNOWS**. It's a *label* as the node's label. It begins with uppercase and use UPPER\_SNAKE\_CASE.
+It's a *relationship* with the *label* **KNOWS**. It's a *label* as the node's label. It use UPPER\_SNAKE\_CASE.
```
Comment (vendethiel) [inline cypher.html.markdown:53]: While we're at it, can you fix `It use` -> `It uses`, please?

**Comment by vendethiel** [comment]
Sorry for the delay, looks good.

**Comment by mribeirodantas** [comment]
There are conflicts here. Could you please resolve them, @encX ?

**Comment by verhovsky** [comment]
I don't agree with this PR. CamelCase can also be considered camel case. It's less common for people to know what Pascal is or its casing rules. The UPPER_SNAKE_CASE makes sense though.

**Pull request closed without merging**